hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/bucket/GetAclBucketHandler.java

```diff
@@ -32,9 +38,41 @@ public class GetAclBucketHandler extends GetAclHandler {
   @CommandLine.Mixin
   private BucketUri address;
 
+  private enum BucketType { SOURCE, TARGET }
+
+  @CommandLine.Option(names = { "--type", "-t" },
+      defaultValue = "SOURCE",
+      description = "Allowed Bucket Type: ${COMPLETION-CANDIDATES}")
+  private BucketType linkBucketType;
```

Comment:
I would suggest change the option to "--follow", "-f".  
If "-f" is not specified, then link bucket ACL itself will be displayed, otherwise source bucket ACL will be displayed. 

Also for this ${COMPLETION-CANDIDATES}, I didn't find how it is processed, but the its content is 

"FILE_SYSTEM_OPTIMIZED,  OBJECT_STORE, LEGACY", which is not what we need.